docs/develop.rst

```diff
@@ -79,8 +79,8 @@ You can also print some useful advice for the developer here, as we do in the ``
 Template Registration
 =====================
 
-Even though you can use bobtemplates without regitration, we shoudl register the template, to allow plonecli and future mrbob version to query for it. The registration is done by adding en Python entry point into the setup.py of bobtemplates.plone or your own custom bobtemplates package and by adding a short method to the bobregistry.py file.
-
+Even though you can use bobtemplates without registration, we should register the template to allow plonecli and future mrbob versions to query for it.
+The registration is done either by adding a Python entry point into the ``setup.py`` of ``bobtemplates.plone`` or by adding your own custom ``bobtemplates`` package with a short method to the ``bobregistry.py`` file.
```

Comment:
this is not the same meaning, please understand the information before changing a complete sentence. I'll corret it in the master before doing the release.